src/AutoFixture.AutoEF.Tests/NavigationPropertyRequestSpecificationTest.cs

```diff
@@ -67,6 +67,50 @@ public void ReturnsTrueWhenPropertyIsCollection([Frozen] IEntityTypesProvider ty
             var result = sut.IsSatisfiedBy(property);
 
             result.Should().BeTrue();
+        }
+
+        [Theory, AutoNSub]
+        public void ReturnsFalseWhenDeclaringTypeNotRegisteredWhenTableNameId([Frozen] IEntityTypesProvider typesProvider, SUT sut)
+        {
+            typesProvider.GetTypes().Returns(new[] { typeof(Boo) });
+            var property = typeof(Far).GetProperty("Boo");
+
+            var result = sut.IsSatisfiedBy(property);
+
+            result.Should().BeFalse();
+        }
+
+        [Theory, AutoNSub]
+        public void ReturnsFalseWhenPropertyTypeNotRegisteredWhenTableNameId([Frozen] IEntityTypesProvider typesProvider, SUT sut)
+        {
+            typesProvider.GetTypes().Returns(new[] { typeof(Far) });
+            var property = typeof(Far).GetProperty("Boo");
+
+            var result = sut.IsSatisfiedBy(property);
+
+            result.Should().BeFalse();
+        }
+
+        [Theory, AutoNSub]
+        public void ReturnsTrueWhenBothTypesRegisteredWhenTableNameId([Frozen] IEntityTypesProvider typesProvider, SUT sut)
+        {
+            typesProvider.GetTypes().Returns(new[] { typeof(Far), typeof(Boo) });
+            var property = typeof(Far).GetProperty("Boo");
+
+            var result = sut.IsSatisfiedBy(property);
+
+            result.Should().BeTrue();
+        }
+
+        [Theory, AutoNSub]
+        public void ReturnsTrueWhenPropertyIsCollectionWhenTableNameId([Frozen] IEntityTypesProvider typesProvider, SUT sut)
+        {
+            typesProvider.GetTypes().Returns(new[] { typeof(Qix), typeof(Boo) });
+            var property = typeof(Boo).GetProperty("Qixes");
+
+            var result = sut.IsSatisfiedBy(property);
+
+            result.Should().BeTrue();

```

Comment:
I'm not sure these tests need to exist - it looks like the only changed functionality is in `IdPropertySetterInterceptor` and `ParentPropertySetterInterceptor`, in which case the `NavigationPropertyRequestSpecification` will not have been touched